static/app/views/settings/projectMetrics/projectMetrics.tsx

```diff
@@ -150,6 +155,22 @@ function ProjectMetrics({project, location}: Props) {
 
       <CardinalityLimit project={project} />
 
+      {hasExtractionRules && (
+        <Fragment>
+          <SearchWrapper style={{marginBottom: space(1)}}>
```

Comment:
Wouldn't we prefer to style the component using CSS-in-JS rather than inline styles?